react/features/recording/functions.js

```diff
@@ -215,7 +215,7 @@ export async function sendMeetingHighlight(state: Object) {
 
     const reqBody = {
         meetingFqn: extractFqnFromPath(state),
-        sessionId: conference.sessionId,
+        sessionId: conference.sessionId || conference.room.meetingId,
```

Comment:
@saghul is this a good way to get the sessionId on mobile?

Reply:
Let me check.

Reply:
```suggestion
        sessionId: conference.getMeetingUniqueId(),
```